<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>show_followers</class>
 <widget class="QWidget" name="show_followers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Followers</string>
  </property>
  <property name="styleSheet">
   <string notr="true">show_followers
{
	background-color:#0a0612;
}
QPushButton
{
	background-color:#8a02b2;
	color:#1f1137;
	border-radius:20px;
}
QPushButton:hover
{
	background-color:#341d5c;
	color: #ffffff;
}
QLabel
{
	color:#ad03de;
}
QLineEdit
{
	background-color:#673ab7;
	color:#ffffff;
	border-radius:16px;
}
QLineEdit:hover
{
	background-color:#5d34a5;
}
QPlainTextEdit
{
	color:#ffffff;
	background-color:#3e236e;
	border-radius:12px;
}</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>521</width>
     <height>741</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QPlainTextEdit" name="pte_follower3">
      <property name="minimumSize">
       <size>
        <width>320</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>320</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="btn_follow5">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Follow</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="btn_follow4">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Follow</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="btn_follow2">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Follow</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QPushButton" name="btn_follow8">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Follow</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPlainTextEdit" name="pte_follower1">
      <property name="minimumSize">
       <size>
        <width>320</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>320</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="btn_follow1">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Follow</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="btn_follow3">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Follow</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="btn_follow6">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Follow</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPlainTextEdit" name="pte_follower2">
      <property name="minimumSize">
       <size>
        <width>320</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>320</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPlainTextEdit" name="pte_follower8">
      <property name="minimumSize">
       <size>
        <width>320</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>320</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="btn_follow7">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Follow</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QPlainTextEdit" name="pte_follower9">
      <property name="minimumSize">
       <size>
        <width>320</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>320</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPlainTextEdit" name="pte_follower6">
      <property name="minimumSize">
       <size>
        <width>320</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>320</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPlainTextEdit" name="pte_follower7">
      <property name="minimumSize">
       <size>
        <width>320</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>320</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPlainTextEdit" name="pte_follower4">
      <property name="minimumSize">
       <size>
        <width>320</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>320</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPlainTextEdit" name="pte_follower5">
      <property name="minimumSize">
       <size>
        <width>320</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>320</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QPlainTextEdit" name="pte_follower10">
      <property name="minimumSize">
       <size>
        <width>320</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>320</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="btn_follow9">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Follow</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QPushButton" name="btn_follow10">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Follow</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>620</y>
     <width>61</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btn_up">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Up</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_down">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Down</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>190</y>
     <width>160</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>60</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>20</y>
     <width>561</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <widget class="QLineEdit" name="le_search">
      <property name="minimumSize">
       <size>
        <width>320</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>320</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_search">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_refresh">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>60</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>19</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
